bigquery/unit_tests/test__helpers.py

```diff
@@ -491,6 +491,363 @@ def test_instance_setter_w_bad_udfs(self):
         self.assertEqual(instance.udf_resources, [])
 
 
+class Test_AbstractQueryParameter(unittest.TestCase):
+
+    @staticmethod
+    def _get_target_class():
+        from google.cloud.bigquery._helpers import AbstractQueryParameter
```

Comment:
Yeah, I think @dhermes has [asked](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/2663#discussion_r86286081) me to in the [past](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/2770#discussion_r90081046). I guess we just need to *commit* to one or the other. (*pun somewhat intended*)